src/main/java/com/maxmind/geoip2/record/Traits.java

```diff
@@ -45,29 +64,19 @@
      * This attribute is only available from the City and Insights web
      * service end points.
      */
+    @JsonProperty("autonomous_system_number")
     public Integer getAutonomousSystemNumber() {
         return this.autonomousSystemNumber;
     }
 
-    /**
-     * @return The organization associated with the registered <a
-     * href="http://en.wikipedia.org/wiki/Autonomous_system_(Internet)"
-     * >autonomous system number</a> for the IP address. This attribute
-     * is only available from the City and Insights web service end
-     * points.
-     */
-    public String getAutonomousSystemOrganization() {
-        return this.autonomousSystemOrganization;
-    }
 
```

Comment:
Any particular reason you reordered these methods? It makes spotting the differences difficult.